test/vtop_example.sh

```diff
@@ -252,7 +252,7 @@ function verifyVtadminSetup() {
 function verifyVTOrcSetup() {
   # Set the primary tablet to readOnly using the vtctld and wait for VTOrc to repair
   primaryTablet=$(getPrimaryTablet)
-  vtctldclient SetReadOnly "$primaryTablet"
+  vtctldclient SetWritable "$primaryTablet" false
```

Comment:
Do we still have SetReadWrite command as well?

Reply:
No we don't. In vtctldclient we only have `SetWritable`.
In vtctlclient we have `SetReadOnly` and `SetReadWrite`